server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds/AppCenterBuilds.cs

```diff
@@ -0,0 +1,92 @@
+using Arcadia.Assistant.AppCenterBuilds.Contracts;
+using Arcadia.Assistant.MobileBuild.Contracts;
+using Microsoft.ServiceFabric.Services.Communication.Runtime;
+using Microsoft.ServiceFabric.Services.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Fabric;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Arcadia.Assistant.AppCenterBuilds
+{
+    /// <summary>
+    /// An instance of this class is created for each service instance by the Service Fabric runtime.
+    /// </summary>
+    public class AppCenterBuilds : StatelessService
+    {
+        private readonly IDownloadApplicationSettings configuration;
+        private readonly IHttpClientFactory clientFactory;
+        private readonly IMobileBuildActorFactory mobileBuildFactory;
+
+        private const int DefaultDownloadBuildIntervalMinutes = 720;
```

Comment:
Put default in a configuration parsing / initialization

Reply:
Done